ecosystem/typescript/sdk/examples/typescript/transactions_management.ts

```diff
@@ -0,0 +1,127 @@
+/**
+ * This example demonstrates how a client can utilize the TransactionWorker class.
+ */
+
+import { AptosAccount, BCS, TxnBuilderTypes, TransactionWorker, FaucetClient, Provider, Types } from "aptos";
+import { exit } from "process";
+import { NODE_URL, FAUCET_URL } from "./common";
+
+const provider = new Provider({ fullnodeUrl: NODE_URL, indexerUrl: NODE_URL });
+
+const faucet = new FaucetClient(NODE_URL, FAUCET_URL);
+
+async function main() {
+  const accountsCount = 5;
+  const transactionsCount = 100;
+  const totalTransactions = accountsCount * transactionsCount;
+
+  console.log("starting...");
+  console.log(new Date().toTimeString());
+  // create senders and recipients accounts
+  const senders: AptosAccount[] = [];
+  const recipients: AptosAccount[] = [];
+  for (let i = 0; i < accountsCount; i++) {
+    senders.push(new AptosAccount());
+    recipients.push(new AptosAccount());
+  }
+  console.log(`${senders.length + recipients.length} sender and recipient accounts created`);
+
+  // funds sender accounts
+  const funds: Array<Promise<string[]>> = [];
+
+  for (let i = 0; i < senders.length; i++) {
+    funds.push(faucet.fundAccount(senders[i].address().noPrefix(), 10000000000));
+  }
+
+  // send requests
+  await Promise.all(funds);
+  console.log(`${funds.length} sender accounts funded`);
+  for (const acc in senders) {
+    const curr = senders[acc] as AptosAccount;
+    console.log(curr.address().hex());
+  }
+
+  // read sender accounts
+  const balances: Array<Promise<Types.AccountData>> = [];
+  for (let i = 0; i < senders.length; i++) {
+    balances.push(provider.getAccount(senders[i].address().hex()));
+  }
+  // send requests
+  await Promise.all(balances);
+
+  console.log(`${balances.length} sender account balances checked`);
+
+  // create transactions
+  const payloads: any[] = [];
+  // 100 transactions
+  for (let j = 0; j < transactionsCount; j++) {
+    // 5 recipients
+    for (let i = 0; i < recipients.length; i++) {
+      const txn = new TxnBuilderTypes.TransactionPayloadEntryFunction(
+        TxnBuilderTypes.EntryFunction.natural(
+          "0x1::aptos_account",
+          "transfer",
+          [],
+          [BCS.bcsToBytes(TxnBuilderTypes.AccountAddress.fromHex(recipients[i].address())), BCS.bcsSerializeUint64(5)],
+        ),
+      );
+      payloads.push(txn);
+    }
+  }
+
+  const batchTransactions = (payloads: TxnBuilderTypes.Transaction[], sender: AptosAccount) => {
+    const transactionWorker = new TransactionWorker(provider, sender);
+    const waitFor: Array<Promise<void>> = [];
+
+    transactionWorker.start();
```

Comment:
it would probably be good to leave a comment here and/or in the start method how this works.